File: hw/santorum/u3/hw29Singleton/Python/model/appointment.py
class Appointment:
    def __init__(self, appointment_id, patient_id, patient_email, date, time):
        self.appointment_id = appointment_id
        self.patient_id = patient_id
        self.patient_email = patient_email
        self.date = date
        self.time = time

    def to_file_string(self):
        return f"{self.appointment_id},{self.patient_id},{self.patient_email},{self.date},{self.time}\n"

File: hw/santorum/u3/hw29Singleton/JavaScript/model/Appointment.js
export default class Appointment {
  constructor(appointmentId, patientId, patientEmail, date, time) {
    this.appointmentId = appointmentId;
    this.patientId = patientId;
    this.patientEmail = patientEmail;
    this.date = date;
    this.time = time;
  }
}
